modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -257,10 +281,22 @@ definition {
 		Click(locator1 = "FormViewBuilder#OPEN_SIDEBAR");
 	}
 
+	macro ruleSave {
+		AssertClick(
+			locator1 = "FormViewBuilder#RULE_BUILDER_SAVE_BUTTON",
+			value1 = "Save");
+	}
+
 	macro save {
 		AppBuilderAdmin.save();
 	}
 
+	macro saveFormView {
```

Comment:
hi @nkkwon , please remove this macro and use FormViewBuilder.save() macro in the test case.